devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy

```diff
@@ -31,6 +31,7 @@ metadata {
 		fingerprint mfr: "0371", prod: "0002", model: "0003", deviceJoinName: "Aeotec NanoMote Quad", mnmn: "SmartThings", vid: "generic-4-button" //EU
 		fingerprint mfr: "0086", prod: "0101", model: "0058", deviceJoinName: "Aeotec KeyFob", mnmn: "SmartThings", vid: "generic-4-button" //US
 		fingerprint mfr: "0086", prod: "0001", model: "0058", deviceJoinName: "Aeotec KeyFob", mnmn: "SmartThings", vid: "generic-4-button" //EU
```

Comment:
Please remove this change from the PR and just fix the manufacturer name

Reply:
just removed,please check again

Reply:
Now the whole file is deleted